### PR TITLE
[BD-27] [TNL-7455] [TNL-7457] Add support for True/False and Fill in the Blank problem types

### PR DESCRIPTION
**Pull request by pcockwell**
Provides conversion support for these question types:

* `cc.true_false.v0p1`
* `cc.fib.v0p1`

This PR also fixes conversion support for `cc.multiple_choice.v0p1` problems, and given that the logic for parsing QTI `cc.true_false.v0p1` problems is identical to the logic for parsing `cc.multiple_choice.v0p1` problems, the code is reused. Similarly, the logic for generating OLX Multiple Choice and OLX True/False questions are identical, and code is reused there as well.

**JIRA tickets**:
[BLENDED-581](https://openedx.atlassian.net/browse/BLENDED-581)
[TNL-7455](https://openedx.atlassian.net/browse/TNL-7455)
[TNL-7457](https://openedx.atlassian.net/browse/TNL-7457)

**Dependencies**: Builds directly upon #5

**Testing instructions**:

Note: Requires Python `3.8.X` to work on my machine

Unit Tests:
1. Create and activate a virtual environment for this project 
2. Install requirements with `make requirements`
3. Run tests with `make test`

Manual Testing:
1. Create and activate a virtual environment for this project 
2. Install requirements with `make requirements`
3. Install the `cc2olx` CLI with `make install`
4. Make note of the install location listed at the end of the install process:
```
...
creating /home/USERNAME/.pyenv/versions/3.6.10/envs/cc2olx/lib/python3.6/site-packages/cc2olx-0.1.0-py3.6.egg
Extracting cc2olx-0.1.0-py3.6.egg to /home/USERNAME/.pyenv/versions/3.6.10/envs/cc2olx/lib/python3.6/site-packages
cc2olx 0.1.0 is already the active version in easy-install.pth
Installing cc2olx script to /home/USERNAME/.pyenv/versions/cc2olx/bin              # <-------- This line

Installed /home/USERNAME/.pyenv/versions/3.6.10/envs/cc2olx/lib/python3.6/site-packages/cc2olx-0.1.0-py3.6.egg
Processing dependencies for cc2olx==0.1.0
Finished processing dependencies for cc2olx==0.1.0
```
5. Find a Common Cartridge course (`.imscc` file) that contains both True/False questions, and Fill in the Blank questions
6. Inspect the course on https://common-cartridge-viewer.netlify.app/#/ to locate where the True/False and Fill in the Blank questions are. You will need to navigate to the same location in the Studio after importing your course
7. Convert the Common Cartridge course with
```
/home/USERNAME/.pyenv/versions/cc2olx/bin/cc2olx -i /path/to/course.imscc -r zip
```
8. This file will be stored in the `output` directory. Navigate to your Studio, select a course, and import the generated `.tar.gz` file to overwrite the course.
9. Navigate to the course outline, and find the sections containing the True/False and Fill in the Blank questions, and inspect the resulting rendering

**Reviewers**
- [ ] @0x29a 
- [ ] edX reviewer[s] TBD

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @pcockwell! I've created [BLENDED-581](https://openedx.atlassian.net/browse/BLENDED-581) to keep track of it in Jira.   More details are on the [BD-27](https://openedx.atlassian.net/wiki/spaces/COMM/pages/1544912903/BD-27+Common+Cartridge+Converter) project page.

When this pull request is ready, tag your edX technical lead.

**Review by 0x29a — APPROVED** [review]
:+1:

Looks good to me.

- [x] I tested that:
- Tests are passing locally for both python versions.
- Fill-in-the blank problems are being rendered correctly:
    Screenshot from 2020-09-16 16-05-25
- True/false problems are being rendered correctly:
    Screenshot from 2020-09-16 16-15-43
- [x] I read through the code
- [ ] ~~I checked for accessibility issues~~
- [x] Includes documentation
- [ ] ~~I made sure any change in configuration variables is reflected in the corresponding client's `configuration-secure` repository.~~

File: src/cc2olx/qti.py [hunk 1]
```diff
@@ -236,12 +273,34 @@ def _parse_problem_profile(self, problem):
...
+            for BOOLEAN type problems
+        """
+        return {
+            MULTIPLE_CHOICE: self._parse_multiple_choice_problem,
+            MULTIPLE_RESPONSE: self._parse_multiple_response_problem,
+            FILL_IN_THE_BLANK: self._parse_fib_problem,
+            ESSAY: self._parse_essay_problem,
+            BOOLEAN: self._parse_multiple_choice_problem,
+            PATTERN_MATCH: self._parse_pattern_match_problem,
+        }
```
[the remark this answers is not in the document]
Comment (azarembok) [reply src/cc2olx/qti.py:293]: 👍

**Review by azarembok — APPROVED** [review]
Looks good, thanks!

**Pull request merged**